src/components/GestureHandlerButton.tsx

```diff
@@ -1,6 +1,13 @@
 import * as React from 'react';
-import { View } from '../ReactCompat';
+import { Insets, View } from '../ReactCompat';
+import { RawButtonProps } from './GestureButtons';
 
-export default React.forwardRef<View>((props, ref) => (
-  <View ref={ref} accessibilityRole="button" {...props} />
-));
+interface RawButtonWebProps extends RawButtonProps {
+  hitSlop?: Insets | undefined;
+}
+
+export default React.forwardRef<View, { rippleColor: any }>(
```

Comment:
If only it was so simple 😞 I've spent a lot of time racking my brain to get rid of this TS error and that was the only solution I've come up with. Well, maybe not the only one, but the other was to add `{rippleColor: any}` into `createNativeWrapper`:

```jsx
const ComponentWrapper = React.forwardRef<
    React.ComponentType<any>,
    P & NativeViewGestureHandlerProps & {rippleColor: any}
  >((props, ref) => {
```

which I really don't like

Reply:
🪄 summon @tjzel, help plz

Reply:
👀

Reply:
I don't have any problem with this:

```tsx
import * as React from 'react';
import { Insets, View } from '../ReactCompat';
import { RawButtonProps } from './GestureButtons';

interface RawButtonWebProps extends RawButtonProps {
  hitSlop?: Insets | undefined;
  rippleColor?: string | undefined;
}

export default React.forwardRef<View, RawButtonWebProps>(
  (props: RawButtonWebProps, ref) => (
    <View ref={ref} accessibilityRole="button" {...props} />
  )
);
```

Where should I look for the error?

Reply:
Well, I do 😞 Here's what I get:

<img width="885" alt="image" src="https://github.com/software-mansion/react-native-gesture-handler/assets/63123542/62c1038d-30da-401a-96ea-070f34f006ea">

This happens when I try to build `Next` app, doing lint check and `tsc --noEmit` in root gives no errors.

Reply:
@m-bert If it's not urgent, let's find an opportunity to work on this offline.

Reply:
@tjzel It is not. Moreover, we already reverted [one of the PRs](https://github.com/software-mansion/react-native-gesture-handler/pull/2896) that was a part of this whole process, so I think it will be the best to start with a discussion about what we want to achieve and which direction should we take.

Reply:
```
MMMMMMMMMMMMMMMMMMMMMMMWNXKKKKKKKKKKXWMMMMMMMMMMMMMMMWWWWMMMMMMM
MMMMMMMMMMMMMMMWX0XWWNXK00KXXXNXXXX0OOKNNWWMMMMMMMMMN0kOKNMMMMMM
MMWXXWMMMMMMMW0dllkKK0OO0KKXXXXXXKK0OO0KKKXXNWMMMMMMW0dxKWMMMMMM
MW0loXMMMMMMWXOxk0KXXKOOOO00000KKKKKKKXXXXKKKKXNNMMMMWKXWMMMMMMM
W0c;oKMMMMMNK0KXXXK00OxdddxkkOOO000KKKXKK0KXXXXK0XWMMMMMMMMMMMMM
Xl',l0WWWWNKO00KK00Okxl::codkkOOOOOO0000OkO0KKK0OOKNWWMMMMMMMMMM
0;..'l0XXXKKKK0OkkOOkkoccdk0KXXKKK0OkkkOOkkkkkO0KKXKXXXNWMMMMMMM
NkoddxOKXXXNNXK00KKXXK0kk0XXXXXXXXK0kkOKXXXK000KXXXNXXK0KNMMMMMM
MMMNOkO0KKXXXKOO0KXXXK0OO0KKKKKKKK0OkxOKK00K00OOKXXXKK0Ok0WMMMMM
MMMXxdxkOOOOOkddkOO0OOxddxkOOOOOOOkxdodkxdxkkkxxxkOOOOkxod0NWMMM
MMMNxcloodddol::clooolc::cllooddoool:;:cloxkkkxdoldddooc:ldONMMM
MMMMKo;;;;;;;,,'',,,,,'''',,;;;;;;,,''',:ldxxddlc;;:;;;,:odxKMMM
WX00KKkl;,''''''''..'''''...........''',:cloolc:,''...';lood0WMM
Nd:ldONWXOxl;'''''.';cc,................',;cc:;,......,::clkNMMM
MKl:lxKMMMMWXOdlc;';kXk;..................,lxkxl'...',;;;:xNMMMM
MMKl:d0KXWMMMMWK0OkO0xc,,,,,;,,,,,,,;;,;;;l0WMWOol:;,;cxOKNMMMMM
MMM0l:;;ckXWMMMMMMMMXxl:,..............,:lONMMMMWWNK00XWMMMMMMMM
MMMMNXx,';lKWMMMMMMNXKKKx'            .okk0KKNWWMMMMMMMMMMMMMMMM
MMMMMMWOodONMWXOOOOo::col:;;;,'...',,,:c:;;;;cox0XWWWMMMMMMMMMMM
MMMMMMMMWXK0Okl;;c:'..'',,,,,;cc:::cllllc:;;,'',:dxkXMMMMMMMMMMM
MMMMMMMW0ol:;''';c:,.',,,,,;;:cllcclllccc::ccc:::lo0WMMMMMMMMMMM
MMMMMMMMNOo:;,..'''..',,,''.',:llllllcccccllc::::lOWMMMMMMMMMMMM
MMMMMMMMMWXx:'......',;;;,,,'.':clooollllolc:cc:cxNMMMMMMMMMMMMM
MMMMMMMMMMMW0c......';;;;;;;;'.';:looooooolcclccckWMMMMMMMMMMMMM
MMMMMMMMMMWXkc......',,,,''',;'.';cloooooolc::cclOWMMMMMMMMMMMMM
MMMMMMMMMNkc'........'',.. .::,..,:cllooool;,:c:oKMMMMMMMMMMMMMM
MMMMMMMN0l;,'........',;.  ......';:clloool'..,:dXMMMMMMMMMMMMMM
MMMMMMNk:;;,'.......',;:,   ....'',;:ccllll;..,co0WMMMMMMMMMMMMM
MMMMMWKo;;;,'......',,;::,....''''',;:cclllc;;cllxXWMMMMMMMMMMMM
MMMMNkl;;,,'......',,;;::;;,,,''''',;;:cccccccclllxXMMMMMMMMMMMM
MMMNOo:,'''.......'',;;;;;;,,,'''''',;;:::cccccccllOWMMMMMMMMMMM
MMMMWNk;'''........',,,,,,,,''''''''',;;;;::cccccccdXMMMMMMMMMMM
MMMMMNd'''.........'',,,,,''.........',;;:clllllllcl0MMMMMMMMMMM
MMMMM0:'''..........'''','...........';:cccccccccccl0MWNMMMMMMMM
MMMMWx,'''............';o:..........';;;;:::c:c:::coKN0KWMWNWMMM
MMMMXo'''......;,.....'lkl.........',,'..,;::::,.';cxkkKWNOONMMM
MMMMNd;,'.....;dc...  .lOk:'......',,....';;::;...':okkxkxdONMMM
MMMMWNK0c....'lkd,..  .;d00Oo,....,;'. .';:::cc,..,cdxoclokXWMMM
MMMMMMMWk,...,lxOxc;;,;;;:ldd:....'::,';cccccllc::cloooodkXWMMMM
MMMMMMMMXl...':ok0XXKK0Oc...'......,:ccccccccclllccoddxOKWMMMMMM
MMMMMMMMMKkko,.;cdk000ko;'..',,,,,'',,;::::cccccoxO0KXNWMMMMMMMM
MMMMMMMMMMMMNx,..',;c:,..''.',;;;;,,,,,;,,,;,,:dKWMMMMMMMMMMMMMM
MMMMMMMMMMMMMNx;,::,',,'..'''',;;;;;;;;;'.''':xXMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMWXKNNKo;,;;,''',;;;;;;;,,';:;,cONMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMWXKKkl;,,,:ccccc:::cc:;l0WMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMN0o:,,,,,,;;,,,,:xXMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMW0d:,,,'''',:d0WMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMW0o:,.'',ckNMMMMMMMMMMMMMMMMMMMMMMM
```